pkg/gitreceive/build.go

```diff
@@ -112,6 +110,20 @@ func build(
 		}
 	}
 
+	_, disableCaching := appConf.Values["DEIS_DISABLE_CACHE"]
```

Comment:
`DEIS_KUBERNETES_DEPLOYMENTS` worked similarly to this, but it's no longer in use, deployments having become the default.